addons/website/static/src/snippets/s_tabs/001.scss

```
@@ -22,4 +22,68 @@
             transform: translateX(1rem);
         }
     }
+
+    .s_tabs_main {
+        --card-cap-bg: initial;
+
+        .nav-tabs{
+            --nav-tabs-link-hover-border-color: transparent;
+        }
+
+        .s_tabs_nav:has(.nav-tabs), .s_tabs_nav:has(.nav-buttons) {
+            background: var(--tabs-bg-color, $gray-100);
+
+            .nav .nav-item a.nav-link{
+                color: var(--tabs-link-color, $body-color);
+            }
+        }
+
+        .nav-tabs a.active {
+            color: $body-color;
+        }
+
+        .nav-underline {
+            border-bottom: 1px solid $border-color;
+
+            .nav-link {
+                margin-bottom: -1px;
+
+                &:not(.active) {
+                    border: 0;
+                }
+            }
+        }
+
+        &.row .nav-underline {
+            @include media-breakpoint-up(sm) {
+                border-bottom: 0;
+                border-left: 1px solid $border-color;
+
+                .nav-link {
+                    margin-left: -1px;
+                    padding: map-get($spacers, 1) map-get($spacers, 3);
+
+                    &.active {
+                        border-left: 2px solid $primary;
```

Normally `$link-color == $primary`, but these variables are customizable and there not may be correlation.
I'd suggest to unsere it by using `$link-color`:

```suggestion
                        border-left: 2px solid $link-color;
```